aws-crt-kotlin/common/test/aws/sdk/kotlin/crt/auth/signing/SigningTest.kt

```diff
@@ -55,182 +54,142 @@ class SigningTest : CrtTest() {
+            normalizeUriPath = true
+            signedBodyValue = AwsSignedBodyValue.EMPTY_SHA256
+            expirationInSeconds = 60
         }
+
+        val signedRequest = AwsSigner.signRequest(request, signingConfig)
+        assertTrue(signedRequest.headers.contains("X-Amz-Date", "20150830T123600Z"), "${signedRequest.headers}")
+        val prefix = "AWS4-ECDSA-P256-SHA256 Credential=AKIDEXAMPLE/20150830/service/aws4_request, SignedHeaders=host;x-amz-date;x-amz-region-set, Signature="
+        assertTrue(signedRequest.headers["Authorization"]!!.contains(prefix), signedRequest.headers["Authorization"])
     }
 
-    @Ignore // FIXME Enable when Kotlin/Native implementation is complete
     @Test
     fun testSigningChunkTrailingHeaders() = runTest {
-        StaticCredentialsProvider.build {
-            accessKeyId = "AKID"
-            secretAccessKey = "SECRET"
-        }.use { provider ->
-
-            val creds = provider.getCredentials()
-
-            val signingConfig = AwsSigningConfig.build {
-                algorithm = AwsSigningAlgorithm.SIGV4
-                signatureType = AwsSignatureType.HTTP_REQUEST_TRAILING_HEADERS
-                region = "foo"
-                service = "bar"
-                date = 1651022625000
-                credentialsProvider = provider
-                credentials = creds
-            }
-
-            val trailingHeaders = Headers.build {
-                append("x-amz-checksum-crc32", "AAAAAA==")
-                append("x-amz-arbitrary-header-with-value", "test")
-            }
-
-            val previousSignature = "106d0654706e3e8dde144d69ca9882ea38d4d72576056c724ba763f8ed3074f3".encodeToByteArray()
-
-            val signature = AwsSigner.signChunkTrailer(trailingHeaders, previousSignature, signingConfig).signature.decodeToString()
-            val expectedSignature = "24f8ed01c7add645b75e65d2382fae5233b97526fdd1a2c4094933b93f6a08bf" // validated using DefaultAwsSigner
-            assertEquals(expectedSignature, signature)
+        val signingConfig = AwsSigningConfig.build {
+            algorithm = AwsSigningAlgorithm.SIGV4
+            signatureType = AwsSignatureType.HTTP_REQUEST_TRAILING_HEADERS
+            region = "foo"
+            service = "bar"
+            date = 1651022625000
+            credentials = Credentials(TEST_ACCESS_KEY_ID, TEST_SECRET_ACCESS_KEY, null)
+        }
+
+        val trailingHeaders = Headers.build {
+            append("x-amz-checksum-crc32", "AAAAAA==")
+            append("x-amz-arbitrary-header-with-value", "test")
         }
+
+        val previousSignature = "106d0654706e3e8dde144d69ca9882ea38d4d72576056c724ba763f8ed3074f3".encodeToByteArray()
+
+        val signature = AwsSigner.signChunkTrailer(trailingHeaders, previousSignature, signingConfig).signature.decodeToString()
+        val expectedSignature = "8b578658fa1705d62bf26aa73e764ac4b705e6d9efd223a2d9e156580f085de4" // validated using DefaultAwsSigner
```

Comment:
Note: the signature changed due to different credentials being used, but I did check again with `DefaultAwsSigner`